src/login/login.ts

```diff
@@ -506,6 +506,8 @@ export function renderSignInPopup (dom: HTMLDocument) {
 
   const loginToIssuer = async (issuerUri: string) => {
     try {
+      // clear authorization metadata from store
+      solidLogicSingleton.store.updater.flagAuthorizationMetadata() as any
```

Comment:
the metadata should be flagged as well on log out

Reply:
It is done lines 711-712

Reply:
https://github.com/SolidOS/solid-ui/blob/b361f8d2efc9a31785cd36aa5e8fc5829dcc9c63/src/login/login.ts#L712

Reply:
ah sorry, I thought it is login as well because github stripped away the context. all good